utils/Test.go

```diff
@@ -15,11 +15,10 @@ func TestHexToFelt(t testing.TB, hex string) *felt.Felt {
 	return f
 }
 
-func TestBigIntToFelt(t testing.TB, big *big.Int) *felt.Felt {
+func TestBigIntToFelt(t testing.TB, big *big.Int) (*felt.Felt, error) {
```

Comment:
But error is always nil, right? 🤔

Reply:
I don't think so, because you can pass in invalid strings ["!InValid:)"]